pkgs/development/libraries/glibc/common.nix

```diff
@@ -349,4 +349,7 @@ stdenv.mkDerivation (
         }
         // (args.meta or { });
     }
+  // lib.optionalAttrs stdenv.hostPlatform.useBolt {
```

Comment:
No need for conditional. (Avoiding mass rebuild is not worth it.)

Reply:
Done